relayer-cli/src/commands/start.rs

```diff
@@ -33,7 +33,7 @@ impl Runnable for StartCmd {
         let config = (*app_config()).clone();
         let config = Arc::new(RwLock::new(config));
 
-        let supervisor_handle = make_supervisor::<ProdChainHandle>(config.clone(), self.full_scan)
+        let supervisor_handle = make_supervisor::<ProdChainHandle>(config, self.full_scan)
```

Comment:
Less cloning of massive data structures, good.

Reply:
Looks like it was cloning an `Arc` though, so supposedly it should have just consisted of incrementing a reference count.

Reply:
It likely does not matter in the startup code, but in general dereferencing an `Arc` is still cheaper than atomically incrementing a ref count.